cpmonitor/templates/index.html

```diff
@@ -29,6 +30,20 @@ <h2 class="card-title">Klimaneutrale Städte und Kommunen</h2>
       </div>
     </div>
   </div>
+  <div class="row row-deck row-cards">
+    {% for city in cities %}
+      <div class="col-sm-6 col-lg-3">
+        <a class="card" href="{% url 'city' city.slug %}">
+          <div class="card-body">
+            <div class="d-flex align-items-center">
+              <div class="subheader">{{ city.name }}</div>
+            </div>
+            {{ city.introduction | truncatechars:100 | safe_markdown }}
```

Comment:
truncating server side by number of chars leads to only a rough estimate of three lines, because of the varying width of characters. This can be done more elegantly using css. See class `.lz-card-summary` in main.scss for an example that is being used to truncate the number of lines for the task cards. We could have two classes `.lz-card-summary-3` and `.lz-card-summary-8` for 3 and 8 lines respectively.